config/routes.rb

```diff
@@ -96,7 +96,11 @@
     resources :tags, except: [:show]
 
     namespace :qa do
-      resources :issues, only: [:index, :show, :edit]
+      resources :issues, only: [:index, :show] do
+        member do
+          patch :update_state
```

Comment:
Can we use the default `:update` route/action?

Reply:
yup, changed to use `updated`